packages/generator/README.md

```diff
@@ -1,11 +1,4 @@
-# generator
+# @leancodepl/contractsgenerator - LeanCode TypeScript Contracts Generator
```

Comment:
I think this is readme that will be placed in main/entry component (also in npm) so it should be basically the same as the README in root dir (maybe it should be replaced and README from main should link to this README?)

Reply:
Makes sense